docs/maintain-guides-how-to-build-validator-community.md

```diff
@@ -0,0 +1,21 @@
+---
```

Comment:
I'm wondering if this would make sense to have this inside another page rather than a separate page itself? Since the material is small, we could place it inside learn-validator or another page for instance. @Swader @lsaether @ansonla3 what do you think?

Reply:
Agreed, this should be a part of the how to be a validator page. We have too many pages in the wiki.